Pod/Classes/NSAttributedString+BonMotUtilities.m

```diff
@@ -0,0 +1,88 @@
+//
+//  NSAttributedString+BonMotUtilities.m
+//  Pods
```

Comment:
#req please replace `Pods` in your headers with `BonMot`. Xcode is annoying, about this, but I updated all the files to match recently, so it would be nice to keep them consistent.